### PR TITLE
Adds a Burn Medkit to Mining Vendor

### PR DESCRIPTION
**Pull request by Mey-Ha-Zah**
## About The Pull Request

Adds a burn medkit to the mining vendor. Since burn damage is slightly harder to deal with on lavaland, I've made it price slightly higher than the brute medkit.
## Why It's Good For The Game

With the addition of vileworms and bileworms, both dealing burn damage, along with the already hazardously flammable nature of lavaland, its about time miners had access to a burn medkit instead of having to use an expensive medipen to heal the damage.
## Changelog
:cl:
add: Burn Medkit to Mining Vendor
/:cl:

### CODE REVIEW

**Comment by improvedname** [comment]
can't you just ordee these in the cargo shuttle, makes use of your money atleast

**Comment by improvedname** [comment]
can't you just ordee these in the cargo shuttle, makes use of your money atleast

**Comment by Pickle-Coding** [comment]
Just go to the department that is designed to treat medical issues and perfectly synergises with mining, medbay.

**Comment by improvedname** [comment]
In my honest opinion we should remove the brute medkit so
1 miners have to go to medbay and interract
2 miners have to use their personal money/cargo budget to order extra supplies besides their mining vendor

**Comment by VladinXXV** [comment]
Gonna be honest with you; I actually agree with the "no fun allowed" take here. I don't find myself using the brute medkit as is, and I wouldn't use a burn medkit either. If my wounds aren't terrible, I'll eat fungi/cacti or get legion cores. If I'm in real bad shape, I'm just going to go to medical. Miners make some serious bank and have little to spend it on, a medkit is a completely rational purchase to make through cargo. And of course, If i'm really so inclined, nothing is stopping me from buying those medkits (or even an advanced medkit) and having robotics make a medibot for me to keep on the mining station.

**Comment by B4CKU** [comment]
> In my honest opinion we should remove the brute medkit so 1 miners have to go to medbay and interract

that does sound a bit ridiculous.
if you were a miner, and you got hit once by a goliath, you'd have to lose few minutes navigating back to the lavaland base, another minute or two to get back to the station, then around a minute going back to the medical department, where you'd wait anywhere from 0 to 15 minutes to get basic first aid, because doctors are to busy trying to resurrect the dead and then you'd still have to go back down to cargo, then lavaland, then to the spot where you got hit in the first place, so you can continue mining.
that sounds more frustrating than anything.

**Comment by VladinXXV** [comment]
> > In my honest opinion we should remove the brute medkit so 1 miners have to go to medbay and interract
> 
> that does sound a bit ridiculous. if you were a miner, and you got hit once by a goliath

If you're a miner and you get hit _once_ by a goliath, you're perfectly fine to keep mining. Get a legion core or eat cacti/leafy mushrooms if you're so concerned.

**Comment by Mey-Ha-Zah** [comment]
I've heard all the arguments for and against this before one way or another. Yes you can buy them via cargo shuttle, in fact converting points to cash, and then buying them is the much cheaper option.
However. Not everyone who ever plays mining is someone with enough hours in the game to know all the contents of the cargo orders, or wants to go to medbay every time they get hit in the back by a vileworm or bileworm. I frequently teach people to convert points to cash to buy more kits in the long run for LONG rounds when its actually feasible to do so, but not every round is an hour or so. Some rounds end 20 minutes in just because chaos does as chaos wills.
Giving people the option to buy a more expensive version of the medkit at a convenience cost gives them the ability to go on with their day if they are in a rush and don't want to wait.
TBH I kinda wanna remove the medipens and replace them with something else more tactile.

**Comment by the-orange-cow** [comment]
Remember guys: healthy miners equates to more ore for the station.

**Comment by jughu** [comment]
Next we gonna add is clone damage medicine because we miners also take [genetic damage] and we will add this to the mining vendor(so we can roll at the greed ruin easier)

**Comment by rasepretep** [comment]
> TBH I kinda wanna remove the medipens and replace them with something else more tactile. I hate medipens.

If/when you do that, do consider that unless the miner in question is ridiculously good at fighting megafauna, luxpens are absolutely essential to fighting it, most notably Bubblegum and Colossus, who can put you into hardcrit if you make the slightest mistake or lag for even a second.

File: code/game/machinery/computer/orders/order_items/mining/order_consumables.dm [hunk 1]
```diff
@@ -13,6 +13,11 @@
 	item_path = /obj/item/storage/medkit/brute
 	cost_per_order = 600
 
+/datum/orderable_item/consumables/medkit_fire
+	item_path = /obj/item/storage/medkit/fire
+	desc = "For emergency magmatic burn relief."
+	cost_per_order = 700
```
Comment (JohnFulpWillard) [inline code/game/machinery/computer/orders/order_items/mining/order_consumables.dm:19]: shouldnt this be the same price as brute?

Comment (Mey-Ha-Zah) [reply code/game/machinery/computer/orders/order_items/mining/order_consumables.dm:19]: I increased the price compared to the brute as sources of burn damage on lavaland: Drake Fire, Lava, etc can lead to burn wounds which can lead to infections etc, while currently, nothing on lavaland can cause you bleeding damage, or broken bones.

Comment (jughu) [reply code/game/machinery/computer/orders/order_items/mining/order_consumables.dm:19]: though burn kit doesn't have mesh to fix said infections maybe at max hercuri which is a nightmare in its own

**Pull request merged**